### DISCUSSION TITLE
Git Branching Model

### DISCUSSION THREAD

**Discussion by aleks-f** [pocoproject/poco#4459] [Ideas]
@obiltschnig @matejk @frwilckens @pavledragisic @Bjoe

**Comment by Bjoe** [comment]
Hi,

here some very quick proposal to start the disscusison:
FreeBSD is doing like this -> https://www.freebsd.org/security/#model
See on git -> https://cgit.freebsd.org/src/
Or maybe a better view -> https://github.com/freebsd/freebsd-src/branches

So there is a "Major version" stable branch like stable/14 branch of from master.
From the "Major version" stabel branch (stable/14) there is a new branch releng/14.2 for the release.

I think we didn't need to have a stable and releng branch. 
But maybe a stable/1.13 branch, branch of from main, with tags for every release. For example 1.13.1 tag... next bug fix would be on the stable/1.13 branch and tag it with 1.13.2 .... etc etc.
For the 1.14.x we have then a stable/1.14 branch, branch of from current main, and a release tag like 1.14.1 ... for the next bug fix on stable/1.14 we will have a 1.14.2 tag on it etc etc.

So that would be my "quick" proposal. Maybe I found some time to create an "diagram" from it to better understand my idea ....
